libeos-updater-util/flatpak-util.c

```diff
@@ -2671,3 +2744,64 @@ euu_flatpak_transaction_uninstall (FlatpakInstallation *installation,
 
   return transaction_run_single_op (transaction, cancellable, error);
 }
+
+
+gboolean
+euu_flatpak_transaction_prune_ref (FlatpakInstallation  *installation,
+                                   const gchar          *remote,
+                                   const gchar          *formatted_ref,
+                                   gboolean              no_prune,
```

Comment:
Could do with a brief comment (maybe formatted as a gtk-doc comment for the function) which explains what `no_prune` means in the context of a function whose name suggests that the only thing it does is to prune.